app/src/pages/Play/operations.graphql

```diff
@@ -29,6 +29,11 @@ mutation EndCurrentTurnAndStartNextTurn(
   $nextTurnSecondsPerTurnOverride: Int
   $roundId: uuid
 ) {
+  delete_turn_scorings(where: { turn_id: { _eq: $currentTurnId }}) {
+    returning {
+      id
+    }
+  }
   insert_turn_scorings(objects: $currentTurnScorings) {
```

Comment:
see: https://hasura.io/docs/1.0/graphql/manual/mutations/update.html#replace-all-nested-array-objects-of-an-object